whitepapers/Dates.md

```diff
@@ -0,0 +1,72 @@
+# Partially Unknown Dates
+
+## Overview
+Many works in SHARE have the wrong publication date displayed, because many sources provide only partial publication dates for some works, such as "May 2016" or just "2016".
+When the day of the month is omitted, the current date parsing logic assumes the first day of the month. When the month is omitted, it assumes January.
+
+### Goals
+Allow storing and displaying works with partially-known publication dates.
+
+## Changes
+
+Create a custom `PartialDate` class with:
+* Three main properties:
```

Comment:
Would it make sense to also go ahead and handle dates that have months and days but not years?